src/components/editor/PropertyResourceTemplate.jsx

```diff
@@ -12,50 +12,65 @@ class PropertyResourceTemplate extends Component {
   constructor(props) {
     super(props)
     this.state = {
-      collapse: false
+      collapse: false,
+      output: this.populatePropertyTemplates()
     }
   }
 
   handleAddClick = (event) => {
-     event.preventDefault()
+    event.preventDefault()
+    const existingOutputs = [...this.state.output]
+
+    existingOutputs.push(<h4 key={shortid.generate()}>{this.props.resourceTemplate.resourceLabel}</h4>)
+    const result = this.populatePropertyTemplates()
+    this.setState({ output: existingOutputs.concat(result)})
   }
 
   handleMintUri = (event) => {
     event.preventDefault()
   }
 
+  populatePropertyTemplates = () => {
+    const newOutput = []
+    this.props.resourceTemplate.propertyTemplates.map((property) => {
+      const keyId = shortid.generate()
+      const newReduxPath = [...this.props.reduxPath]
+      newReduxPath.push(keyId)
+      newOutput.push(<PropertyTemplateOutline
+                      propertyTemplate={property}
+                      rtId={this.props.resourceTemplate.id}
+                      reduxPath={newReduxPath}
+                      key={keyId} />)
+    })
+    return newOutput
+  }
+
   render() {
+    const isAddDisabled = this.props.isRepeatable == "false"
```

Comment:
- `false` does not equal `"false"` so ... `false`
- `"false"` does equal `"false"` so ... `true`
- `true` does not equal `"false"` so ... `false`
- `"true"` does not equal `"false"` so ... `false`

not sure this the result you want, either.

Reply:
New function `isRepeatable` in Utilities address these combinations and returns the expected boolean along with tests in the latest commit.